dev/AppNotifications/AppNotifications.idl

```diff
@@ -111,6 +111,29 @@ namespace Microsoft.Windows.AppNotifications
 
         // Gets or sets whether a Notification's pop-up UI is displayed on the user's screen.
         Boolean SuppressDisplay;
+
+        // Gets or sets the Notification Device Data
+        AppNotificationDevicesData DevicesData;
+    }
+
+    // The Notification Device Data
+    [contract(AppNotificationsContract, 2)]
+    runtimeclass AppNotificationDevicesData
+    {
+        // Initializes a new Instance of NotificationDevicesData
+        AppNotificationDevicesData();
+
+        // Checks if Video or Audio Calling is supported
+        static Boolean IsVideoOrAudioCallingSupported();
```

Comment:
Consider splitting this into two methods - one for audio, one for video?

What will a developer do with this information?

Reply:
We are introducing two new elements on toast:
![image](https://github.com/user-attachments/assets/0b30481a-c006-43b6-85ef-2308362f6167)

This API is to check if these two elements are available in OS or not. Both would be available together.

Can we divide into two APIs like below?
1. IsCameraPreviewAvailable()
2. IsDevicesControlSettingsAvailable()

Reply:
@jonwis , this API will let the user know if this feature is supported by the User's OS version.

can we change to "IsEnhancedCallingSupported"?

Reply:
Consider `IsCallingPreviewSupported`

Reply:
Modified in latest commit